styles/card/_styles.scss

```diff
@@ -87,29 +87,29 @@
     .flag {
         @include tap-highlight-color(transparent);
         align-self: flex-start;
+        color: $flag-on-fill-color;
         flex-shrink: 0;
-        height: $flag-height;
-        margin-bottom: $flag-margin-bottom;
-        margin-right: $flag-margin-right;
-        margin-top: $flag-padding-top;
-        width: $flag-width;
 
         &:hover {
             cursor: pointer;
         }
 
-        &:active {
-            filter: none;
-        }
+        &.on {
+            @include icon-block-with-size($fa-var-flag-o, $icon-size-large);
```

Comment:
this is wrong...in on state it should be icon $fa-var-flag instead of $fa-var-flag-0